docs/src/components/landing-page/TwitterCard.vue

```diff
@@ -0,0 +1,39 @@
+<template>
+  <div v-html="cardContent">
```

Comment:
Unless incredibly rare scenarios, when you're using `v-html` you got something wrong, and this scenario doesn't seem to be an exception

Reply:
I went this way so I I could very easily place the content in an array an loop through them. I didn't really see this as bad since the data therein is not coming from a user and is static. I'll make the whole carousel a component (so it handles everything twitter carousel related), getting rid of v-html means I don't really need the twitter component anyways.

Reply:
> getting rid of v-html means I don't really need the twitter component anyways.
This is the main reason I said something is probably wrong

Having that code into a variable won't get you html highlight, wrong html tag closing detection, etc etc
Not bad per-se, but strips away a lot of safenets which the IDE provides you out of the box

Reply:
roger that.